README.md

```diff
@@ -7,47 +7,25 @@ TextWorld requires __Python 3__ and only supports __Linux__ and __Mac__ systems
 
 ### Requirements
 
-The following system libraries are required to install and run TextWorld:
-```
-sudo apt-get -y install uuid-dev libffi-dev build-essential curl gcc make python3-dev
-```
-as well as some Python libraries that can be installed separately using
-```
-pip install -r requirements.txt
-```
+TextWorld requires some system libraries for its native components.
+On a Debian/Ubuntu-based system, these can be installed with
+
+    sudo apt install build-essential uuid-dev libffi-dev python3-dev curl
```

Comment:
Should we also provide the command for installing it on OSX?

Reply:
I thought we were installing those for Travis?

Reply:
Ah right.  They're already installed on the Travis macOS images, so I don't have them in the config.  I'll figure the names out, one second.